devices/Hx711/GainLevel.cs

```diff
@@ -14,22 +14,23 @@ public enum GainLevel : byte
 
         /// <summary>
         /// Gain factor not set.
+        /// If use for class initialize gives ArgumentOutOfRangeException exception.
```

Comment:
This explanation belongs in the `Scale` comment for exception where it has context.
In this context its _just_ "Gain factor not set".

Reply:
Seems I've updated everything as necessary buffers and variables.
[here is changes](https://github.com/nanoframework/nanoFramework.IoT.Device/pull/451/commits/b3f57fe2d093422566c44beb9541d6016cbb2aa1)

But I'm not sure if it become visible or not.

Reply:
@lsd-techno they are! A few last comments from Laurent and this should be OK.

Reply:
@josesimoes done